bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/handler/VeluxBridgeHandler.java

```diff
@@ -382,11 +401,11 @@ private void bridgeParamsUpdated() {
 
         logger.trace("bridgeParamsUpdated(): Fetching existing scenes.");
         bridgeParameters.scenes.getScenes(thisBridge);
-        logger.info("Found {} scenes:\n\t{}", VeluxBindingConstants.BINDING_ID,
+        logger.info("Found Velux scenes:\n\t{}",
```

Comment:
Ok. 
Note: I think this was originally there so that users could see the details of auto-discovered actuators needed to create the respective `.things` file. But with the OH3 paradigm, and the auto-discovery, this information is visible in the Inbox, so a `logger.info()` is no longer required.